src/python/grpcio_tests/commands.py

```diff
@@ -41,6 +40,7 @@
 import traceback
 
 import setuptools
+from distutils import errors
```

Comment:
`distutils` is a standard Python library, so this import should be right back up before `glob`. Let's try not to run away from the style guide too much when it's easy not to.